src/constants.js

```diff
@@ -43,12 +52,14 @@ export const GOOGLE_LOGIN_CLIENT_ID =
 
 export const GOOGLE_APIS_CLIENT_ID =
 	process.env.GOOGLE_APIS_CLIENT_ID ||
-	'249789152860-12uce9nrsb6s0k582epuk92bbjsm8bm7.apps.googleusercontent.com';
+	'249789152860-g6d9o3e7prmt98qipm3aucisemk70q5l.apps.googleusercontent.com';
 
 export const GOOGLE_PROJECT_ID =
 	process.env.GOOGLE_PROJECT_ID || 'stuyactivities-org';
 
+	// delete this later
```

Comment:
?

Reply:
Windows user didn't know how to run with process variables